docs/getstdhandle.md

```diff
@@ -77,6 +84,16 @@ Some applications operate outside the boundaries of their declared subsystem; fo
 
 Some applications may also vary their behavior on the type of inherited handle. Disambiguating the type between console, pipe, file, and others can be performed with [**GetFileType**](/windows/win32/api/fileapi/nf-fileapi-getfiletype).
 
+### Handle disposal
+
+It is not required to [**CloseHandle**](/windows/win32/api/handleapi/nf-handleapi-closehandle) when done with the handle retrieved from **GetStdHandle**. The returned value is simply a copy of the value stored in the process table. The process itself is generally considered the owner of these handles and their lifetime. It is placed in the table on creation depending on the inheritance and launch specifics of the [**CreateProcess**](/windows/win32/api/processthreadsapi/nf-processthreadsapi-createprocessw) call and will be freed when the process is destroyed.
+
+The case to manipulate the lifetime of these handles manually with **CloseHandle** is if intentionally trying to replace or block other parts of the process from using these handles going forward. `HANDLE` is a numerical value, so another thread storing it locally will not necessarily be updated when the value from the table is updated with **SetStdHandle**. Closing the value will close it process wide and the next usage of the stored copy will encounter an error on **ReadFile** or **WriteFile** accordingly.
+
+Guidance for replacing a standard handle in the process table would be to get the existing `HANDLE` from the table with **GetStdHandle**, use **SetStdHandle** to place a new `HANDLE` in that is opened with **CreateFile** (or a similar function), then to close the retrieved handle.
```

Comment:
Can you reword this? "Guidance would be to" is so passively-voiced off might be falling my head

Reply:
yeah ok tried again